### ISSUE TITLE
Action outage?

### ISSUE THREAD

**Issue by aaronadamsCA** [Azure/functions-action#100]
At some point between 15:42 and 16:05 UTC, all attempts to run this action began throwing an error immediately on invocation:

```
Error: File not found: '/home/runner/work/_actions/Azure/functions-action/v1/lib/main.js'
```

**Comment by ryancole** [comment]
I am also seeing this issue.

**Comment by patelchandni** [comment]
It is fixed in latest v1 (v1.4.2)
